benchmarks/src/main/java/com/squareup/benchmarks/gson/FromJsonBenchmark.kt

```diff
@@ -0,0 +1,28 @@
+package com.squareup.benchmarks.gson
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import com.squareup.benchmarks.FromJsonBenchmarkBase
+import com.squareup.benchmarks.gson.model.Response
+import org.openjdk.jmh.annotations.Benchmark
+import org.openjdk.jmh.annotations.Level
+import org.openjdk.jmh.annotations.Setup
+import java.io.InputStreamReader
+
+open class FromJsonBenchmark : FromJsonBenchmarkBase() {
```

Comment:
flatten this and its base? Typically with JMH you combine benchmarks with parameters, not subtyping

Reply:
... you’d have `@Benchmark` methods like fromJsonWithGson, fromJsonWithMoshi, etc.